### ISSUE TITLE
Randomizing sounds for alphanumeric keys

### ISSUE THREAD

**Issue by kosivantsov** [aitjcize/Qwertickle#3]
I think it would sound more realistic if there were several similar sounds for alphanumeric keys and they would be assigned to groups or picked at random.
Here's a project where this random pickup is done:

https://github.com/colszowka/linux-typewriter

**Comment by rwmpelstilzchen** [comment]
+1, but the sound files should be carefully selected. `linux-typewriter` is nice and minimalist, but the sound it produces is cacophonous due to the sound files.

**Comment by kosivantsov** [comment]
yes, I agree, but if there's a settings dialog where a user can select his/her sounds, alternative sounds can be left for the user's own discernment . By default there can be only one sound, just as it is now.
